ngraph/core/src/op/transpose.cpp

```diff
@@ -42,6 +42,7 @@ void op::v1::Transpose::validate_and_infer_types() {
 
     set_input_is_relevant_to_shape(1);
 
+    NGRAPH_SUPPRESS_DEPRECATED_START
```

Comment:
Why we add deprecations to the methods which are used in existing opset operations? Do we have a replacement for them?

Reply:
We will remove it from only public API, just move to private part